### PR TITLE
NWN2: Change cursor appropriately during mouse-over of a creature

### PR DESCRIPTION
**Pull request by rjshae**

### CODE REVIEW

File: src/engines/nwn2/cursor.h [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+ */
+
+#ifndef ENGINES_NWN2_CURSOR_H
+#define ENGINES_NWN2_CURSOR_H
+
+namespace Engines {
+
+namespace NWN2 {
+
+/**
```
Comment (DrMcCoy) [inline src/engines/nwn2/cursor.h]: This is not a Doxygen comment, so please use just plain `/*` here.

File: src/engines/nwn2/object.cpp [hunk 2]
```diff
@@ -300,7 +303,7 @@ uint32 Object::getFaction() const {
 /** Return this object's reputation with the source. */
 uint8 Object::getReputation(Object *source) const {
 	assert(source);
-	return 0; // Default to 'enemy'
+	return getArea()->getModule().getFactions().getReputation(source, getFaction());
```
Comment (DrMcCoy) [inline src/engines/nwn2/object.cpp]: getArea() might return 0, so please check before dereferencing

**Comment by rjshae** [comment]
Changes applied. Thanks.

**Comment by DrMcCoy** [comment]
Merged as c81b917b0...08c5cf7fa, thanks! :)

**Pull request closed without merging**